src/views/Dashboard/Customer/Home/GeneticData/RequestAnalysis/index.vue

```diff
@@ -98,19 +98,15 @@ export default {
 
       if (!dataList) return this.isEmpty = true
 
-      for (let i = 0; i < dataList.length; i++) {
-        const geneticData = await queryGeneticDataById(this.api, dataList[i])
-        let { id, owenerId, reportLink, title, description, createdAt, updatedAt } = geneticData
-        
-
+      for (let { id, owenerId, reportLink, title, description, createdAt, updatedAt } of dataList ) {
```

Comment:
Why `let` not `const`?

Reply:
because if updateAt is not 0, I changed the value of createdAt to show updated date in table

Reply:
because I need to reassign the value of createdAt .But turns out it becomes minor bugs in sonar cloud . Should I use var instead or ? @agustinustheo